### PR TITLE
Bump ui_test

### PR DESCRIPTION
**Pull request by oli-obk**
This makes `ui_test` parse `--bless` and allows a follow up change to use `Mode::Error` (instead of `Mode::Yolo`) with `RustfixMode::Everything`

changelog: none

### CODE REVIEW

**Comment by Alexendoo** [comment]
Opened https://github.com/oli-obk/ui_test/pull/157 to get `OutputConflictHandling::Error` back, particularly for the internal tests where the command is difficult to remember

File: tests/compile-test.rs [hunk 1]
```diff
@@ -302,8 +291,7 @@ fn run_ui_cargo() {
 
     ui_test::run_tests_generic(
         vec![config],
-        args,
-        |path, args, _config| path.ends_with("Cargo.toml") && ui_test::default_filter_by_arg(path, args),
+        |path, config| path.ends_with("Cargo.toml") && ui_test::default_file_filter(path, config),
```
Comment (Alexendoo) [inline tests/compile-test.rs:294]: This ought to be `default_any_file_filter` (or whatever I called it), the one that doesn't require `.rs`

Comment (oli-obk) [reply tests/compile-test.rs]: whoops fixed

**Comment by Alexendoo** [comment]
Thanks! @bors r+

**Pull request merged**